agent/http-server.c

```diff
@@ -61,28 +61,32 @@ hwangsae_http_server_set_recording_dir (HwangsaeHttpServer * server,
 }
 
 static gchar *
-get_file_path (gchar * base_path, gchar * record_id, gchar * ext)
+get_file_path (gchar * base_path, gchar * file_id, gchar * ext)
 {
+  g_autofree gchar *filename = NULL;
   gchar *file_path;
 
-  file_path =
-      g_strdup_printf ("%s/hwangsae-recording-%s-00000.%s", base_path,
-      record_id, ext);
+  filename = g_strdup_printf ("hwangsae-recording-%s.%s", file_id, ext);
```

Comment:
As we discussed, I assume that the filename would have `start timestamp` and `duration`(or  `stop timestamp`).
Does `file_id` have the information?

Reply:
Current file name schema
File name: hwangsae-recording-[record_id]-[start_timestamp]-[end_timestamp].[ext]
Based on this schema 
file_id: [record_id]-[start_timestamp]-[end_timestamp]

This PR will be updated based on the work splitted on PR #37 #38.